src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/FulfilmentSurveyEmailTemplateEndpoint.java

```diff
@@ -59,6 +59,12 @@ public FulfilmentSurveyEmailTemplateEndpoint(
   public List<EmailTemplateDto> getAllowedEmailTemplatesBySurvey(
       @RequestParam(value = "surveyId") UUID surveyId,
       @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    authUser.checkUserPermission(
+            userEmail,
+            survey.getId(),
```

Comment:
`survey` hasn't been defined or populated yet here. This could be changed to directly use the surveyId from the request params instead.